concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+Zhou-2020-350-12	12	belt	腰帶	772	BELT
+Zhou-2020-350-13	13	big	大	1202	BIG
+Zhou-2020-350-14	14	bitter	苦	887	BITTER
+Zhou-2020-350-15	15	black	黑	163	BLACK
+Zhou-2020-350-16	16	blood	血	946	BLOOD
+Zhou-2020-350-17	17	boat	船	1844	BOAT
+Zhou-2020-350-18	18	boiling water	開（水）	3362	BOILED WATER
+Zhou-2020-350-19	19	bottom	底下	834	BUTTOCKS
+Zhou-2020-350-20	20	bovine	牛	3044	BOVINE
+Zhou-2020-350-21	21	bowl	碗	1530	BOWL
+Zhou-2020-350-22	22	bran	糠	2362	BRAN
+Zhou-2020-350-23	23	bright	亮	679	BRIGHT
+Zhou-2020-350-24	24	bug	蟲	1027	BUG
+Zhou-2020-350-25	25	buy	買	1869	BUY
+Zhou-2020-350-26	26	cat	貓	1208	CAT
+Zhou-2020-350-27	27	cave	洞	1155	CAVE
+Zhou-2020-350-28	28	cedar tree	杉樹		
+Zhou-2020-350-29	29	chilly	涼		
+Zhou-2020-350-30	30	Chinese	漢	3373	HAN CHINESE
+Zhou-2020-350-31	31	chopsticks	筷子	1987	CHOPSTICKS
+Zhou-2020-350-32	32	cl. for animals	（一）頭（牛）		
+Zhou-2020-350-33	33	clf. for clothes	件		
+Zhou-2020-350-34	34	clf. for general objects	個		
+Zhou-2020-350-35	35	cloth	布	1616	CLOTH
+Zhou-2020-350-36	36	clothes	衣服	1895	CLOTHES
+Zhou-2020-350-37	37	cold	冷	1287	COLD
+Zhou-2020-350-38	38	comb	梳子	1475	COMB
+Zhou-2020-350-39	39	crow	烏鴉	2074	CROW (VERB)
+Zhou-2020-350-40	40	daughter-in-law	兒媳	1057	DAUGHTER-IN-LAW
+Zhou-2020-350-41	41	day	（一）天	1260	DAY (24 HOURS)
+Zhou-2020-350-42	42	deaf	聾	996	DEAF
+Zhou-2020-350-43	43	dew	露水	1977	DEW
+Zhou-2020-350-44	44	diluted	稀		
+Zhou-2020-350-45	45	direction	方向	3427	DIRECTION
+Zhou-2020-350-46	46	divide	分	1359	SEPARATE
+Zhou-2020-350-47	47	dragon	龍	2038	DRAGON
+Zhou-2020-350-48	48	drill	鑽子	3423	DRILL
+Zhou-2020-350-49	49	drink	喝	1401	DRINK
+Zhou-2020-350-50	50	drum	鼓	908	DRUM
+Zhou-2020-350-51	51	drunk	醉	1463	DRUNK
+Zhou-2020-350-52	52	dry	乾	1398	DRY
+Zhou-2020-350-53	53	dustpan	簸箕	2051	DUSTPAN
+Zhou-2020-350-54	54	eagle	老鷹	1905	EAGLE
+Zhou-2020-350-55	55	ear of grain	穗子	3244	EAR (OF GRAIN)
+Zhou-2020-350-56	56	earth	地	1228	EARTH (SOIL)
+Zhou-2020-350-57	57	egg	蛋	744	EGG
+Zhou-2020-350-58	58	eye, blind	眼睛、盲	1248	EYE
+Zhou-2020-350-59	59	face	臉	1560	FACE
+Zhou-2020-350-60	60	fan	扇子	1971	FAN (OBJECT)
+Zhou-2020-350-61	61	fat (adj.)	肥	1279	FAT (OBESE)
+Zhou-2020-350-62	62	feces	屎	675	FAECES (EXCREMENT)
+Zhou-2020-350-63	63	feet	腳	1301	FOOT
+Zhou-2020-350-64	64	female	雌性	2260	FEMALE (OF PERSON)
+Zhou-2020-350-65	65	female genital	女陰		
+Zhou-2020-350-66	66	few	少	1242	FEW
+Zhou-2020-350-67	67	field	田地	212	FIELD
+Zhou-2020-350-68	68	finger	指頭	1303	FINGER
+Zhou-2020-350-69	69	fire	火	221	FIRE
+Zhou-2020-350-70	70	firewood	柴	10	FIREWOOD
+Zhou-2020-350-71	71	fish	魚	227	FISH
+Zhou-2020-350-72	72	five	五	493	FIVE
+Zhou-2020-350-73	73	flat	平	1633	FLAT
+Zhou-2020-350-74	74	flea	蝨子	232	FLEA
+Zhou-2020-350-75	75	flower/blossom	花	239	FLOWER
+Zhou-2020-350-76	76	four	四	1500	FOUR
+Zhou-2020-350-77	77	front	前面	2194	FRONT (PART)
+Zhou-2020-350-78	78	fruit	果子	1507	FRUIT
+Zhou-2020-350-79	79	god, ghost	神、鬼	53	SPIRIT
+Zhou-2020-350-80	80	good	好	1035	GOOD
+Zhou-2020-350-81	81	gourd	葫蘆	411	GOURD
+Zhou-2020-350-82	82	granary	倉庫	3025	BARN OR SHED
```

Comment:
I'm not sure if the gloss refers to BARN or 3657 GRANARY. @LinguList Could you disambiguate?

Reply:
It translates literally as "food store". I'd say, granary is closer.